pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -12,19 +11,19 @@ theme: responsive
 
 The
 %a{href:"https://hourofcode.com"} Hour of Code
-is a global movement reaching tens of millions of students in 180+ countries. Anyone, anywhere can organize an Hour of Code event or try any of the over 200+ one-hour tutorials, available in over 45 languages. Teachers, to view lesson plans and additional teacher resources please select your desired Hour of Code tutorial from the side menu to the left.
+is a global movement reaching tens of millions of students in 180+ countries. Anyone, anywhere can organize an Hour of Code event or try any of the over 200+ one-hour tutorials, available in over 45 languages.
```

Comment:
Do we need to loc this?

Reply:
yeah, i'm going to do a follow-up that localizes everything on this page since there are some other stray PRs i still have open right now